apps/console/src/features/identity-providers/data/identity-provider-templates/templates/saml-identity-provider/saml-idp-wizard-file-based-help.tsx

```diff
@@ -39,8 +40,48 @@ const SAMLIdPWizardFileBasedHelp: FunctionComponent<Props> = (props: Props): Rea
     const { [ "data-testid" ]: testId } = props;
     const config: ConfigReducerStateInterface = useSelector((state: AppState) => state.config);
 
+    const { t } = useTranslation();
+    const { getLink } = useDocumentation();
+
     return (
         <div data-testid={ testId }>
+            <Message info>
+                <p>
```

Comment:
Lets use `Text` component from react-components here.

Reply:
Having `Text` component alters the design slightly. Therefore kept `p` tags as it is for the time being.